### ISSUE TITLE
Support for dotted links

### ISSUE THREAD

**Issue by Anachron** [knsv/mermaid#26]
Sometimes it would be really useful to use dotted links to specify a weak link or temporary break.
Syntax could be:
`A-.->B;`

**Comment by knsv** [comment]
Will look at this now

**Comment by knsv** [comment]
An implementation for this is in place and will be included in next release. The solution uses the following syntax:
```
dotted link: 
-.->

dotted link with text: 
-. text .-> 

thick link: 
==>

think link with text: 
== text ==> 

```

**Comment by cerdman** [comment]
hey @knsv - yep sorry just saw this - that is as far as I got (grammar/syntax) and was pretty much the same - how did you get the style figured out?

**Comment by knsv** [comment]
Fixed with release 0.3.2

**Comment by Anachron** [comment]
Awesome, thanks! @knsv
